routes.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func getCentres(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	data, err := db.Centres.GetAll()
+
+	b, err := json.Marshal(data)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte(`{"error": "error marshalling data"}`))
```

Comment:
The error return value of `w.Write` is not checked.

Better to give it proper error handling just in case something goes wrong and an error is returned.